src/Compilers/CSharp/Portable/Symbols/NativeIntegerTypeSymbol.cs

```diff
@@ -2,16 +2,21 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
+
 using System;
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Diagnostics;
+using Microsoft.CodeAnalysis.PooledObjects;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     internal sealed class NativeIntegerTypeSymbol : WrappedNamedTypeSymbol
```

Comment:
>NativeIntegerTypeSymbol [](start = 26, length = 23)

It feels like we need to be able to retarget these types. Do we handle that?

Reply:
Added a PROTOTYPE comment for now.

---
In reply to: [382305226](https://github.com/dotnet/roslyn/pull/41728#discussion_r382305226) [](ancestors = 382305226)